udata/harvest/tasks.py

```diff
@@ -13,8 +13,9 @@ def harvest(self, ident):
     log.info('Launching harvest job for source "%s"', ident)
 
     source = HarvestSource.get(ident)
-    if source.deleted:
-        return  # Ignore deleted sources
+    if source.deleted or not source.active:
+        log.info('Ignoring inactive source "%s"', ident)
```

Comment:
The log info is incorrect for deleted sources?
```suggestion
        log.info('Ignoring inactive or deleted source "%s"', ident)
```